changelog/8141.feature.rst

```diff
@@ -0,0 +1 @@
+Added `--tmp-path-retention-count` and `--tmp-path-retention-policy` options to control how directories created by the `tmp_path` fixture are kept.
```

Comment:
We should also mention explicitly that the default has changed to remove all directories if all tests passed.

Reply:
Done
[ce71d7b](https://github.com/pytest-dev/pytest/pull/10442/commits/ce71d7b6982f6d32d808e83f0f7f329735d4065b)